src/polyswarmtransaction/transaction.py

```diff
@@ -148,3 +148,24 @@ def __import_transaction_class(module: ModuleType, class_name: str) -> Type[Tran
             raise exceptions.UnsupportedTransactionError(f'{module.__name__}:{class_name} is not a Transaction')
 
         return transaction
+
+
+@dataclasses.dataclass
+class CustomTransaction(Transaction):
+    raw_body: str = None
```

Comment:
When is this used?

Reply:
Right now, never.

By prividing raw_body you could replace the whole message to be signed, not only _data_  key. I was not sure if customizing just _data_ would be enough for us.  

I am not testing this one and can remove if is not useful in your opinion.

Reply:
Oh, so singing blobs

Reply:
I think we want to keep the format we have. 

The name thing I now realize was kind of pointless. But we need the address inside to show the signature matches the signed address inside.

Reply:
The name thing was to not allow same msg to be used on two REST paths, right? I think it will be needed if we want to have a single point of input for Eth transactions

Anyway, I removed the raw_body on ca69a48